### PR TITLE
Support Mir's event recorder

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR:

- updates Mir version
- introduces an environment variable `EUDICO_INTERCEPTOR_OUTPUT` to enable Mir's event recorder (interceptor)
- modifies Circle CI config to be able to download event logs
- removes Tendermint test files

### CODE REVIEW

File: chain/consensus/mir/manager.go [hunk 1]
```diff
@@ -143,6 +153,19 @@ func NewManager(ctx context.Context, addr address.Address, api v1api.FullNode) (
 		return nil, fmt.Errorf("failed to create crypto manager: %w", err)
 	}
 
+	// Instantiate an interceptor.
+
+	var interceptor *eventlog.Recorder
+
+	interceptorOutput := os.Getenv(InterceptorOutputEnv)
```
Comment (adlrocha) [inline chain/consensus/mir/manager.go:160]: What is the interceptor? Should this only be started in "debugging mode" to avoid unnecessary overheads?

Comment (adlrocha) [reply chain/consensus/mir/manager.go:160]: Nevermind, I see below that this is the case already :)

**Review by adlrocha — APPROVED** [review]
LGTM. Merging time :)

File: chain/consensus/mir/manager.go (hunk 1 above)
Comment (dnkolegov) [reply chain/consensus/mir/manager.go:160]: Yes, I also added that to the PR description.
I was trying to get an interceptor from the context but it is changed when Subnet is started, so I decided to use environment variables

**Pull request merged**